pkg/gzip/gzip.go

```diff
@@ -50,32 +45,40 @@ func ExtractTarGz(owner string, fp io.Reader, fpName string, dest string) (err e
 			break
 		}
 		if err != nil {
-			return util.Errorf("Unable to read %s: %s", fpName, err)
+			return util.Errorf("read error: %s", err)
```

Comment:
not sure what you mean by "limit."  The only reason for `fpName` to exist as a parameters is so it can be added to error messages.  That seemed wrong to me, so I moved the file-name-printing out of the method.